src/main/resources/logback.xml

```diff
@@ -25,13 +25,9 @@
         </encoder>
     </appender>
 
-    <root level="DEBUG">
+    <root level="INFO">
```

Comment:
It's not biggie one way or the other but probably best to leave it at `debug` for development.